config/default.yml

```diff
@@ -778,9 +778,10 @@ RSpec/VariableName:
   SupportedStyles:
     - snake_case
     - camelCase
+  AllowedPatterns: []
   IgnoredPatterns: []
```

Comment:
should we remove the obsolete configuration from the default?

Reply:
Obsolete parameters are still available.
It seems to me that it would be better to keep both available for backward compatibility until the complete removal of the corresponding options at https://github.com/rubocop/rubocop, which will come someday.

Reply:
In the meantime, is there a good way of suppressing this on every run?
```
Warning: obsolete parameter `IgnoredPatterns` (for `RSpec/VariableName`) found in vendor/bundle/ruby/3.1.0/gems/rubocop-rspec-2.13.2/config/default.yml
`IgnoredPatterns` has been renamed to `AllowedPatterns`.
Warning: obsolete parameter `IgnoredPatterns` (for `RSpec/VariableName`) found in .rubocop.yml
`IgnoredPatterns` has been renamed to `AllowedPatterns`.
```

Reply:
Can you provide .rubocop.yml?

Reply:
```
Warning: obsolete parameter `IgnoredPatterns` (for `RSpec/VariableName`) found in .rubocop.yml
`IgnoredPatterns` has been renamed to `AllowedPatterns`.
```

This warning can be suppressed by modifying .rubocop.yml as follows.

### before
```yaml
RSpec/VariableName:
  IgnoredPatterns: [...]
```

### after
```yaml
RSpec/VariableName:
  AllowedPatterns: [...] 
```

Reply:
```
Warning: obsolete parameter `IgnoredPatterns` (for `RSpec/VariableName`) found in vendor/bundle/ruby/3.1.0/gems/rubocop-rspec-2.13.2/config/default.yml
`IgnoredPatterns` has been renamed to `AllowedPatterns`.
```

This is a guess, but I think the following settings may be included in the .rubocop.yml file.

```yaml
inherit_from: vendor/bundle/ruby/3.1.0/gems/rubocop-rspec-2.13.2/config/default.yml
```

Reply:
Yeah, I get the warnings with our .rubocop.yml like:

```yml
inherit_gem:
  rubocop-rspec:
    - config/default.yml

require:
  - rubocop-rspec

AllCops:
  TargetRubyVersion: 3.1
  NewCops: enable
```

I still get the warnings if I append this to the file - 

```yaml
RSpec/VariableName:
  AllowedPatterns: [] 
```

However, looking into it - I'm not so sure we need the inherit_gem bit. Seems like if I remove it we still get the rspec defaults, but don't get the deprecation warnings.

Sorry for the noise!